Google.Ads.Gax/tests/TestUtilities/Google.Ads.Gax.TestUtilities.csproj

```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
   <!-- build properties -->
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;net472;net5.0;net6.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.1;net472;net6.0</TargetFrameworks>
```

Comment:
Should this also have `net8.0` like all the others?

Reply:
Done.